src/plugins/esql_datagrid/kibana.jsonc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "plugin",
+  "id": "@kbn/esql-datagrid",
+  "owner": "@elastic/kibana-esql",
+  "plugin": {
+    "id": "esqlDataGrid",
+    "server": false,
+    "browser": true,
+    "requiredPlugins": [
+      "data",
+      "uiActions",
+      "fieldFormats"
+    ],
+    "requiredBundles": [
+      "kibanaReact",
+      "kibanaUtils",
+      "dataViews",
```

Comment:
I can't seem to find any import from this plugin (other than `import type`). Do we need required bundle?

Reply:
Hmmm, let me check amigo

Reply:
It complains without it